packages/toolpad-app/src/toolpad/AppEditor/PageEditor/UrlQueryEditor.tsx

```diff
@@ -19,6 +20,38 @@ export interface UrlQueryEditorProps {
   pageNodeId: NodeId;
 }
 
+interface UrlQueryStringProps {
+  input: [string, string][] | undefined;
+}
+
+function UrlQueryString({ input }: UrlQueryStringProps) {
+  const [queryString, setQueryString] = React.useState('');
+
+  const createQueryString = React.useCallback(() => {
+    return input?.reduce(
+      (accumulator, fieldAndValue: string[], currentIndex: number) => {
+        const [field, value] = fieldAndValue;
+        return `${accumulator}${currentIndex > 0 ? '&' : ''}${field}=${value}`;
+      },
+      input.length ? '?' : '',
+    );
+  }, [input]);
+
+  React.useEffect(() => {
+    setQueryString(createQueryString() as string);
+  }, [createQueryString, input]);
+
+  return (
+    <Typography
+      sx={{
+        marginTop: '20px',
+      }}
+    >
+      <code>{queryString}</code>
```

Comment:
How does this overflow horizontally? What does this look like if I add e.g. 25 parameters?

Reply:
As it stands, it wraps to the next line. Should we leave it or should we truncate?
![horizontal-overflow-page-params](https://github.com/mui/mui-toolpad/assets/52700465/e9c9d906-d20d-403a-88da-93e227c6cc9c)